internal/utils/test/test.go

```diff
@@ -103,6 +103,11 @@ func RealmServerURL() string {
 	return defaultRealmServerURL
 }
 
+// ExpiredAccessToken returns an expired access token to use for testing
+func ExpiredAccessToken() string {
+	return os.Getenv("BAAS_MONGODB_EXPIRED_ACCESS_TOKEN")
```

Comment:
can we update this to follow the pattern used with `RealmServerURL()`?

basically, it forces to write tests that are aware of the environment restrictions they require, its fairly explicit about what's needed (from an environment standpoint) to get successful test execution.

the main difference with this implementation is we won't have a `defaultExpiredAccessToken`, and instead we are fine to just return `""` (you are going to keep the check for `""` in `SkipUnlessExpiredAccessTokenPresent`)